pkg/vulnsrc/govulndb/govulndb.go

```diff
@@ -49,6 +50,18 @@ func (t *transformer) TransformAdvisories(advisories []osv.Advisory, entry osv.E
 		return nil, oops.With("vuln_id", entry.ID).With("aliases", entry.Aliases).Wrapf(err, "json decode error")
 	}
 
+	var oses, arches []string
+	for _, affected := range entry.Affected {
+		if affected.Package.Name != "stdlib" {
+			continue
+		}
+
+		for _, imp := range affected.EcosystemSpecific.Imports {
```

Comment:
I haven't found any cases where multiple `imports` contained different OS/architectures.

But if that happens - it's better to show a false positive than to miss a vulnerability.